### ISSUE TITLE
How to upgrade the webrtc dependency to latest version

### ISSUE THREAD

**Issue by ivanchy** [open-webrtc-toolkit/owt-client-native#583]
Can i upgrade the 'src/third_party/webrtc' to a new chromium revision, seem this project is using a 2020's old version.

**Comment by Meonardo** [comment]
I think you can upgrade it(change the `DEPS` [file](https://github.com/open-webrtc-toolkit/owt-client-native/blob/ae73d99dd2e33a7b743829068efab4bab3b80c52/DEPS#L333)), but it would not be build successfully(?), 
the `src/third_party/webrtc` is from `owt-deps-webrtc` [repo](https://github.com/open-webrtc-toolkit/owt-deps-webrtc), they did a lot of modifications of the source code.

**Comment by ivanchy** [comment]
yes ， i had tried change the DEPS file，it cannot build success. Anyone know how to modify the newest version to suit it?
